AuditEventConverter throws NPE if AuditEvent is created with a key and no value for data

The Spring AuditEvent can be constructed with just a 'key' instead of 'key=value'. If that is done, the AuditEventConverter throws a NPE when converting the data to strings.

results.put(key, object.toString());  // NPE

Here is the top of a stack trace:
Caused by: java.lang.NullPointerException: null
	at com.newwave.cmsdata.config.audit.AuditEventConverter.convertDataToStrings(AuditEventConverter.java:72) ~[classes/:na]
	at com.newwave.cmsdata.repository.CustomAuditEventRepository$1.add(CustomAuditEventRepository.java:54) ~[classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_51]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[na:1.8.0_51]

--- a/app/templates/src/main/java/package/config/audit/_AuditEventConverter.java
+++ b/app/templates/src/main/java/package/config/audit/_AuditEventConverter.java
@@ -78,8 +78,10 @@ public class AuditEventConverter {
                     WebAuthenticationDetails authenticationDetails = (WebAuthenticationDetails) object;
                     results.put("remoteAddress", authenticationDetails.getRemoteAddress());
                     results.put("sessionId", authenticationDetails.getSessionId());
-                } else {
+                } else if (object != null) {
                     results.put(key, object.toString());
+                } else {
+                    results.put(key, "null");
                 }
             }
         }

